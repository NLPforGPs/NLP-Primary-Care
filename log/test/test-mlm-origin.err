cpu-bind=MASK - bp1-gpu016, task 4294967295 4294967295 [0]: mask 0x101 set

CondaValueError: prefix already exists: /user/home/zu20361/.conda/envs/NLP_GP

[nltk_data] Downloading package wordnet to
[nltk_data]     /user/home/zu20361/nltk_data...
[nltk_data]   Package wordnet is already up-to-date!
[nltk_data] Downloading package punkt to
[nltk_data]     /user/home/zu20361/nltk_data...
[nltk_data]   Package punkt is already up-to-date!
Some weights of the model checkpoint at microsoft/BiomedNLP-PubMedBERT-base-uncased-abstract were not used when initializing BertForMaskedLM: ['cls.seq_relationship.weight', 'cls.seq_relationship.bias']
- This IS expected if you are initializing BertForMaskedLM from the checkpoint of a model trained on another task or with another architecture (e.g. initializing a BertForSequenceClassification model from a BertForPreTraining model).
- This IS NOT expected if you are initializing BertForMaskedLM from the checkpoint of a model that you expect to be exactly identical (initializing a BertForSequenceClassification model from a BertForSequenceClassification model).
2021-12-08 10:22:50,612 - /user/home/zu20361/NLP-Primary-Care/./run_plms.py[line:149] - INFO: Predicting...
2021-12-08 10:22:50,690 - /user/home/zu20361/.conda/envs/NLP_GP/lib/python3.9/site-packages/datasets/builder.py[line:377] - WARNING: Using custom data configuration default-a2d0abb6534a5099
0 examples [00:00, ? examples/s]2021-12-08 10:22:50,909 - /user/home/zu20361/.cache/huggingface/modules/datasets_modules/datasets/transcript_evaldataset/4c048f6bc1501180d0c57dc4d2b8c0d9958946e6a0943b3c5b80a90f1d04b330/transcript_evaldataset.py[line:59] - INFO: Generating examples...
                                  0%|          | 0/1 [00:00<?, ?it/s]100%|██████████| 1/1 [00:00<00:00, 247.13it/s]
2021-12-08 10:22:51,041 - /user/home/zu20361/NLP-Primary-Care/./run_plms.py[line:168] - INFO: MLM method.....
  0%|          | 0/1 [00:00<?, ?ba/s]100%|██████████| 1/1 [00:02<00:00,  2.12s/ba]100%|██████████| 1/1 [00:02<00:00,  2.13s/ba]
/user/home/zu20361/NLP-Primary-Care/nn_model/desc_classifier.py:132: FutureWarning: Using a non-tuple sequence for multidimensional indexing is deprecated; use `arr[tuple(seq)]` instead of `arr[seq]`. In the future this will be interpreted as an array index, `arr[np.array(seq)]`, which will result either in an error or a different result.
  logits = logits[index]

real	1m23.735s
user	0m44.726s
sys	0m29.371s
