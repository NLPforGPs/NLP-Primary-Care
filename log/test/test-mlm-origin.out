JOB ID: 242080
Working Directory: /user/home/zu20361/NLP-Primary-Care
Warning: you have pip-installed dependencies in your environment file, but you do not list pip itself as one of your conda dependencies.  Conda may not use the correct pip to install your packages, and they may end up in the wrong place.  Please add an explicit pip dependency.  I'm adding one for you, but still nagging you.
Start Time: Wed Dec 8 10:22:31 GMT 2021
Downloading and preparing dataset description_dataset/default (download: Unknown size, generated: 2.07 MiB, post-processed: Unknown size, total: 2.07 MiB) to /user/home/zu20361/.cache/huggingface/datasets/description_dataset/default-a2d0abb6534a5099/0.0.0/4c048f6bc1501180d0c57dc4d2b8c0d9958946e6a0943b3c5b80a90f1d04b330...
Dataset description_dataset downloaded and prepared to /user/home/zu20361/.cache/huggingface/datasets/description_dataset/default-a2d0abb6534a5099/0.0.0/4c048f6bc1501180d0c57dc4d2b8c0d9958946e6a0943b3c5b80a90f1d04b330. Subsequent calls will reuse this data.
classification_report:
                 precision    recall  f1-score   support

          blood       0.17      0.88      0.29         8
 cardiovascular       0.38      0.81      0.52        32
      digestive       0.55      0.77      0.64        44
            ear       0.80      0.73      0.76        11
      endocrine       0.50      0.25      0.33        24
            eye       0.30      0.60      0.40         5
         female       0.47      0.50      0.48        14
        general       0.04      0.43      0.07        14
           male       0.40      0.86      0.55         7
musculoskeletal       0.73      0.82      0.77        65
   neurological       0.27      0.55      0.36        20
      pregnancy       0.43      0.27      0.33        11
  psychological       0.66      0.76      0.70        50
    respiratory       0.56      0.78      0.65        37
           skin       0.79      0.69      0.73        32
     urological       0.57      0.72      0.63        18

      micro avg       0.41      0.69      0.51       392
      macro avg       0.47      0.65      0.51       392
   weighted avg       0.55      0.69      0.59       392
    samples avg       0.44      0.73      0.52       392

f1_score 0.5145379414998191
roc_auc 0.8608854140871396
End Time: Wed Dec 8 10:23:55 GMT 2021
